OBAKit/UI/MVVM/OBAStaticTableViewController.m

```diff
@@ -334,14 +336,31 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
     }
 }
 
-- (BOOL)tableView:(UITableView *)tableView canEditRowAtIndexPath:(NSIndexPath *)indexPath {
-    OBABaseRow *row = [self rowAtIndexPath:indexPath];
-    return row.rowActions.count > 0;
-}
-
-- (NSArray<UITableViewRowAction *> *)tableView:(UITableView *)tableView editActionsForRowAtIndexPath:(NSIndexPath *)indexPath {
+- (UISwipeActionsConfiguration *)tableView:(UITableView *)tableView trailingSwipeActionsConfigurationForRowAtIndexPath:(NSIndexPath *)indexPath {
     OBABaseRow *row = [self rowAtIndexPath:indexPath];
-    return row.rowActions;
+    UISwipeActionsConfiguration *actionsConfiguration = [row.rowActions copy];
+    
+    // Add a delete action if a delete model exists.
+    if (row.deleteModel) {
+        UISwipeActionsConfiguration *configurationWithDeleteAction;
+        
+        UIContextualAction *deleteAction = [UIContextualAction contextualActionWithStyle:UIContextualActionStyleDestructive title:OBAStrings.delete handler:^(UIContextualAction *action, UIView *sourceView, void (^completionHandler)(BOOL)) {
+            
+            BOOL deleteStatus = [self deleteRowAtIndexPath: indexPath];
+            completionHandler(deleteStatus);
+        }];
+        
+        if (!actionsConfiguration) {
```

Comment:
you don't need to change this, but I generally find it clearer to put the 'positive' case first. i.e. "if an `actionsConfiguration` object exists, then do this, otherwise do something else," instead of "if an `actionsConfiguration` object DOES NOT exist..."

When it comes to readability, there's a lot riding on an easily-missed `!`

Reply:
I agree with you, updated.